packages/lego-bricks/src/components/Accordion/index.tsx

```diff
@@ -0,0 +1,93 @@
+import cx from 'classnames';
+import { useEffect, useRef, useState } from 'react';
+import styles from './Accordion.module.css';
+
+type TriggerProps = {
+  onClick: () => void;
+  disabled: boolean;
+  open: boolean;
+  rotateClassName: string;
+};
+
+type Props = {
+  disabled?: boolean;
+  defaultOpen?: boolean;
+  animated?: boolean;
+  wrapperClassName?: string;
+  triggerPosition?: 'over' | 'under';
+  Trigger: React.ComponentType<TriggerProps>;
```

Comment:
Why the capital T? Should we call this `triggerComponent` to match the Dropdown and Popover component?

Reply:
Whop should've checked how those components named it - updating now

Reply:
Realized that I can change the variable name, but it has to be capitalized within the Accordion component as only capitalized variables can be used as JSX elements. (`<triggerComponent />` is not allowed)